### ISSUE TITLE
broken aria-menu in the top right nav

### ISSUE THREAD

**Issue by susanev** [chef/automate#3842]
## overview
the container holding the projects menu and user menu in the top nav has a broken aria-menu that is causing accessibility requirements to fail.

## associated code
https://github.com/chef/automate/blob/master/components/automate-ui/src/app/page-components/navbar/navbar.component.html#L60

### What It Means
An ARIA menu does not contain required menu items. An element with role="menu" does not contain at least one element with role="menuitem", role="menuitemcheckbox", or role="menuitemradio".

### Why It Matters
ARIA menus are application menus (like those used in software menu) with a specific keyboard interactions. They are NOT for navigation links on a web site and must contain at least one menuitem, menuitemcheckbox, or menuitemradio element.

### How to Fix It
Ensure that the menu is an application menu and has the appropriate keyboard interactions (menu items are navigated via the arrow keys, not via the Tab key) and internal menu items, otherwise remove the menu role.
